sdk/entra/WebJobs.Extensions.AuthenticationEvents/src/WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -0,0 +1,213 @@
+
+
+# Custom Authentication Extension for Azure Functions
```

Comment:
We should remove this in favor of the one outside the `src` directory.

Reply:
Done